dist/toggle-button-group/toggle-button-group.css

```diff
@@ -9,7 +9,7 @@
     --row-list-toggle-buttons-sm: 2;
     --row-list-toggle-buttons-md: 3;
     --row-list-toggle-buttons-xl: 5;
-    --row-gallery-toggle-buttons-min: 1;
+    --row-gallery-toggle-buttons-min: 2;
```

Comment:
Is this breaking possibly?

Reply:
Nah, it was actually never used. I hooked it up. It was using the min layout one, which has had 2 all along, so this basically makes it so that nothing should change, but it's hooked up to the correct one so we can make adjustments as needed using the css prop.